interface/modules/custom_modules/oe-module-payroll/table.sql

```diff
@@ -0,0 +1,11 @@
+
+#IfNotTable module_payroll_data
+CREATE TABLE IF NOT EXISTS `module_payroll_data` (
+    `id` int NOT NULL,
+    `userid` int DEFAULT NULL,
+    `percentage` decimal(4,2) DEFAULT NULL,
+    `flat` decimal(4,2) DEFAULT NULL
+);
```

Comment:
need to also state the engine of the table which is innodb (see other table inserts for examples)